activity.c

```diff
@@ -1226,6 +1226,37 @@ struct activity filesystem_act = {
 	.bitmap		= NULL
 };
 
+/* HBA usage activity */
+struct activity hba_act = {
+	.id		= A_HBA,
+	.options	= AO_NULL,
+	.magic		= ACTIVITY_MAGIC_BASE,
+	.group		= G_DISK,
+#ifdef SOURCE_SADC
+	.f_count_index	= 10,	/* wrap_get_hba_nr() */
+	.f_count2	= NULL,
+	.f_read		= wrap_read_hba,
+#endif
+#ifdef SOURCE_SAR
+	.f_print	= print_hba_stats,
+	.f_print_avg	= print_avg_hba_stats,
+#endif
+#ifdef SOURCE_SADF
+	.f_render	= render_hba_stats,
+	.f_xml_print	= xml_print_hba_stats,
+	.f_json_print	= json_print_hba_stats,
+	.hdr_line	= "HBA;HBAtxframes;HBArxframes",
```

Comment:
Please use the same metrics names than those displayed by sar (ie. "txframes/s" and not "HBAtxframes")